### DISCUSSION TITLE
Getting Required Library Not Found error on Windows 10

### DISCUSSION THREAD

**Discussion by aviatorrob06** [lavalink-devs/lavaplayer#43] [Q&A] [answered]
Hello- I am not sure if you can help me with this issue but I am using lavaplayer in a Minecraft mod I am making, and I am currently trying to port my mod from Fabric to Forge. It works fine in Fabric, but for some reason in Forge whenever I try to use it it fails to load a native library. I've attached my error log. All I've noticed is that it is not successfully copying the necessary native files to my system's tmp directory, I am not sure why however. I should note this log I've attached was generated when I was using an older version of LavaPlayer, specifically another fork, but this exact error still occurs with this specific updated version.
[log.txt](https://github.com/lavalink-devs/lavaplayer/files/13072471/log.txt)

**Comment by topi314** [comment]
from what I can tell this seems to be an issue with forge, I have no idea how forge mods get bundled but this might be the issue

lavaplayer extracts the natives from the resouces, make sure these are in the mod jar

**Comment by aviatorrob06** [comment, marked as answer]
It looks like including the files directly into the mod worked, however now I face a new error with the loader I am using- so I'll follow up with the developer of said loader. Thanks for the help!
